var/spack/repos/builtin/packages/zlib-ng/package.py

```diff
@@ -77,7 +77,7 @@ def pretend_gcc(self):
             filter_file(r"^gcc=0$", "gcc=1", join_path(self.configure_directory, "configure"))
 
     def configure_args(self):
-        args = []
+        args = ["--with-please-remove-me"]
```

Comment:
Since the argument is asking politely:
```suggestion
        args = []
```